_episodes/03-configuration.md

```diff
@@ -3,91 +3,158 @@ title: "Configuration"
 teaching: 0
 exercises: 0
 questions:
-- "What is user configuration file and how can I use it?"
+- "What is the user configuration file and how can I use it?"
 objectives:
 - "Understand the data directories structure"
 - "Configure ESMValTool to ignore some settings"
 keypoints:
-- "The ``config-user.yml`` file tells ESMValTool what data are input"
-- "The ``config-user.yml`` file tells ESMValTool what directory is the destination"
+- "The ``config-user.yml`` tells ESMValTool about input, output and your preference."
+- " ``rootpath`` determines root directory for input data."
+- " ``output_dir`` is the destination directory."
 ---
 
 ## The configuration file
 
 The ``config-user.yml`` configuration file contains all the global level information needed by ESMValTool to run.
-This is an (YAML file) [https://yaml.org/spec/1.2/spec.html]. An example configuration file can be found in the root directory of the ESMValTool repository.
-Make a copy and rename it to ``config-user.yml``:
+This is an [YAML file](https://yaml.org/spec/1.2/spec.html). An example configuration file can be found in the root directory of the ESMValTool repository.
+Let's change our working directory to ESMValTool, then make a copy of the file and rename it to ``config-user.yml``, then run a text editor called Nano to open it:
 
-~~~
+~~~bash
+  cd ESMValTool
   cp config-user-example.yml config-user.yml
```

Comment:
My workflow is to copy this to a new directory next to ESMValTool called run, or workspace. I find this works better in keeping things in one place. 

Is there a best practices document somewhere that we should consult?

Reply:
@ledm Thanks. I like your suggestion. I couldn't find any best practices for this issue. However, I have checked some lessons. There is no standard way, some choose "code", "data" directories in Desktop which is not good. I applied this workflow: 

```
- esmvaltool_tutorial
   - ESMValTool (Github repo)
   - data
   - other files like a copy of config file. 
```

Instruction on how to make these directories will come in `setup` section of the tutorial.

Reply:
There is an open issue about making config-user.yml more user friendly: https://github.com/ESMValGroup/ESMValCore/issues/64